pkgs/tools/text/transifex-client/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, buildPythonApplication, fetchPypi
+, python-slugify, requests, urllib3 }:
+
+let
+  # Requires an older urllib3 version
```

Comment:
Does this package really require a different version of `urllib3` or is it just a version range check that could be patched?

Reply:
Seems to work when I patched it, managed to pull changes in a project with the patched one. Force pushed.